indico/MaKaC/PDFinterface/conference.py

```diff
@@ -240,55 +231,46 @@ def __init__(self, abstract, tz=None):
 
     @staticmethod
     def _get_status(abstract):
-        status = abstract.getCurrentStatus()
-        #style = ParagraphStyle({})
-        #style.firstLineIndent = -90
-        #style.leftIndent = 90
-        if isinstance(status, review.AbstractStatusDuplicated):
-            original = status.getOriginal()
-            return _("DUPLICATED ({0}: {1})").format(original.getId(), original.getTitle())
-        elif isinstance(status, review.AbstractStatusMerged):
-            target = status.getTargetAbstract()
-            return _("MERGED ({0}: {1})").format(target.getId(), target.getTitle())
+        state_title = abstract.state.title.upper()
+        if abstract.state == AbstractState.duplicate:
+            return _("{} ({}: {})").format(state_title, abstract.duplicate_of.friendly_id, abstract.duplicate_of.title)
+        elif abstract.state == AbstractState.merged:
+            return _("{} ({}: {})").format(state_title, abstract.merged_into.friendly_id, abstract.merged_into.title)
         else:
-            return AbstractStatusList.getInstance().getCaption(status.__class__).upper()
+            return abstract.state.title.upper()
 
     @staticmethod
     def _get_track_judgements(abstract):
```

Comment:
well, if I start renaming legacy methods and classes, this will never end :stuck_out_tongue: 
I just went for adapting the necessary code for it to work. It's gonna be rewritten after 2.0.